src/endpoint/s3/ops/s3_put_bucket_notification.js

```diff
@@ -20,6 +24,23 @@ async function put_bucket_notification(req) {
         }
     }
 
+    //test new notifications.
+    //if test notification fails, fail the put op
+    let config_fs = null;
+    if (req.object_sdk.nsfs_config_root) {
+        config_fs = new ConfigFS(req.object_sdk.nsfs_config_root);
+    }
+    const err = await notif_util.test_notifications(topic_configuration,
+        config_fs?.connections_dir_path);
+    if (err) {
+        throw new S3Error({
```

Comment:
Why not `throw new S3Error(S3Error.InvalidArgument);`? (use existing object)
Why did you add the `detail` field? where do we see it? (could you add a print screen of what the user will see?)

Reply:
I just figures the error would be useful. Looks like this:

aprinzse:noobaa-core$ s3nsfsapi put-bucket-notification-configuration --bucket notif --notification-configuration file:////home/aprinzse/work/standalone/put_conf/put_configuration.json 
urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host 'localhost'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings

An error occurred (InvalidArgument) when calling the PutBucketNotificationConfiguration operation: "ENOENT: no such file or directory, open '/etc/noobaa.conf.d/connections/connect_http.json'"